### ISSUE TITLE
IID should be generated differently

### ISSUE THREAD

**Issue by ra-at-diladele-com** [ra-at-diladele-com/qlproxy_external#1008]
```
var startTime = time.Now().SecondsFrom1970
startTime << 32

int32 counter = 0;

generateNewIid() {
    return startTime | counter++
}

```

**Comment by ea-at-diladele-com** [comment]
Possibly in the next version. Currently it does not really bring any advantages.

**Comment by ra-at-diladele-com** [comment]
done in feature-new-iid, not sure if this will be included into final release of 6.3
